mockstore/mocktikv/mock_tikv_test.go

```diff
@@ -128,502 +100,551 @@ func (s *testMockTiKVSuite) mustDeleteOK(c *C, key string, startTS, commitTS uin
 		PrimaryLock:  []byte(key),
 		StartVersion: startTS,
 	}
-	errs := s.store.Prewrite(req)
+	errs := store.Prewrite(req)
 	for _, err := range errs {
-		c.Assert(err, IsNil)
+		assert.Nil(t, err)
 	}
-	err := s.store.Commit([][]byte{[]byte(key)}, startTS, commitTS)
-	c.Assert(err, IsNil)
+	err := store.Commit([][]byte{[]byte(key)}, startTS, commitTS)
+	assert.Nil(t, err)
 }
 
-func (s *testMockTiKVSuite) mustScanOK(c *C, start string, limit int, ts uint64, expect ...string) {
-	s.mustRangeScanOK(c, start, "", limit, ts, expect...)
+func mustScanOK(t *testing.T, store MVCCStore, start string, limit int, ts uint64, expect ...string) {
+	mustRangeScanOK(t, store, start, "", limit, ts, expect...)
 }
 
-func (s *testMockTiKVSuite) mustRangeScanOK(c *C, start, end string, limit int, ts uint64, expect ...string) {
-	pairs := s.store.Scan([]byte(start), []byte(end), limit, ts, kvrpcpb.IsolationLevel_SI, nil)
-	c.Assert(len(pairs)*2, Equals, len(expect))
+func mustRangeScanOK(t *testing.T, store MVCCStore, start, end string, limit int, ts uint64, expect ...string) {
+	assert := assert.New(t)
```

Comment:
This kind of code introduce the problem `Variable 'assert' collides with imported package name` while it is not quite more convenient than `assert.Xxx(t, )`. I'd prefer avoid this kind of code but not insist.

Reply:
Normally it is a bad idea. But for this special case, I guess it is ok because methods/functions of them are almost the same. In this way, we create a function overloading like effect. By the way, readme of `stretchr/testify` also suggests use `assert`.

Reply:
Fine. I don't have a strong feeling.